forward_engineering/alterScript/alterScriptHelpers/alterEntityHelper.js

```diff
@@ -155,7 +155,17 @@ const getIndexesBasedOnNewlyCreatedColumnsScript = ({_, ddlProvider, dbVersion,
         return []
     }
 
-    return getModifyIndexesScriptDtos({ _, ddlProvider })({ collection, dbVersion })
+    const {getSchemaNameFromCollection} = require('../../utils/general')(_);
+    const additionalDataForDdlProvider = {
```

Comment:
seems like it's a copy\paste from [here](https://github.com/hackolade/PostgreSQL/blob/main/forward_engineering/alterScript/alterScriptHelpers/entityHelpers/indexesHelper.js#L323-L327). Can we move this "data reshaper" to a separate function and reuse in both places?